sys/include/net/eui64.h

```diff
@@ -83,6 +83,66 @@ static inline void eui64_clear_group(eui64_t *addr)
     addr->uint8[0] &= ~EUI64_GROUP_FLAG;
 }
 
+/**
+ * @brief Generates an short address for the netdev interface.
+ *
+ * @note It is possible to supply a board-specific, constant addres
+ *       by implementing @ref board_get_eui_short.
+ *       If no such function is availiable, this will fall back to
+ *       @ref luid_get_short.
+ *
+ * @param[in] netdev    The network device for which the address is
+ *                      generated.
+ *                      Will be passed on to @ref board_get_eui48.
+ * @param[out] addr     The generated short address
+ *
+ */
+void eui_short_get(netdev_t *netdev, uint16_t *addr);
+
+/**
+ * @brief Generates an EUI-64 address for the netdev interface.
+ *
+ * @note It is possible to supply a board-specific, constant addres
+ *       by implementing @ref board_get_eui64.
+ *       If no such function is availiable, this will fall back to
+ *       @ref luid_get_eui64.
+ *
+ * @param[in] netdev    The network device for which the address is
+ *                      generated.
+ *                      Will be passed on to @ref board_get_eui64.
+ * @param[out] addr     The generated EUI-64 address
+ *
+ */
+void eui64_get(netdev_t *netdev, eui64_t *addr);
+
+/**
+ * @brief Board-specific function to supply a short address to a netdev
+ *
+ *        Implement this function in your board code if the board
```

Comment:
```suggestion
 * @note  Implement this function in your board code if the board
```

Reply:
Added the `@note` and a `@warning` not to call the function directly.